aws/resource_aws_cloudwatch_event_rule.go

```diff
@@ -49,6 +50,12 @@ func resourceAwsCloudWatchEventRule() *schema.Resource {
 				Optional:     true,
 				ValidateFunc: validation.StringLenBetween(0, 256),
 			},
+			"event_bus_name": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				ForceNew:     true,
+				ValidateFunc: validateCloudWatchEventBusName,
```

Comment:
Maybe don't validate `event_bus_name` here as that would exclude specifying `default` which is a valid event bus name when manipulating events. We'll then need to suppress the difference between `""` and `"default"`.